docs/docs/api-reference/starlark-reference/toleration.md

```diff
@@ -8,11 +8,33 @@ can be used with a [ServiceConfig][service-config] object.
 
 ```python
 toleration = Toleration(
```

Comment:
```suggestion
toleration = [
Toleration(..)
]
```

Reply:
Hey! This is an individual toleration. It is in a list in ServiceConfig!